web/themes/custom/novel/novel.theme

```diff
@@ -326,29 +326,3 @@ function novel_preprocess_views_view_unformatted(&$variables): void {
     }
   }
 }
-
```

Comment:
Awesome that you move functionality away from the theme :)

Reply:
Thanks 😁